packages/lib/src/components/internal/ClickToPay/components/CtPLogin/CtPLoginInput.tsx

```diff
@@ -57,6 +57,11 @@ const CtPLoginInput = (props: CtPLoginInputProps): h.JSX.Element => {
         [props.onPressEnter]
     );
 
+    const handleOnKeyPress = useCallback((event: h.JSX.TargetedKeyboardEvent<HTMLInputElement>) => {
```

Comment:
Do you need _separate_ handlers for `onKeyPress` and onKeyUp` to handle the user pressing the `Enter` key?
Couldn't it all be handled in the `handleOnKeyPress` function

Reply:
The `onKeyPress` seems to be the only event that can properly prevent the form of being submitted when pressing Enter. I tried the `onKeyUp` and it didn't work.

Perhaps we could try it out to use only `onKeyPress` instead of `onKeyUp`, but that is a big change and would impact all the fields. If we go for it, it is better try that in a different PR

Reply:
ok